src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Dataflow/ReferenceSource/ValueNode.cs

```diff
@@ -77,7 +80,7 @@ public ValueNode ()
 		/// applied so that each 'unique value' can be considered on its own without regard to the structure that led to
 		/// it.
 		/// </summary>
-		public UniqueValueCollection UniqueValues {
+		public UniqueValueCollection UniqueValuesInternal {
```

Comment:
Just curious - why was this renamed to `UniqueValuesInternal` ?

Reply:
I wanted to ask about that one, but I'm trying to hold back on questions that don't matter in the end so that there's not so much feedback. But I'm curious too. @vitek-karas?

Reply:
Because there's also an extension method called `UniqueValues` which is public. All external callers should use the method, because it can handle `null` as the input value (it's an extension method, so `null` `this` is possible and handled). This property is only so that it's possible to implement the method.

This caused bugs in the data flow code since there were couple of places where by accident we called the property instead of the method, and thus would end up with a NullRef if the value happened to be null (which can happen in some special cases).

The name is not ideal - but better than before.